evoting/service/state.go

```diff
@@ -0,0 +1,85 @@
+package service
+
+import (
+	"math/rand"
+	"sync"
+	"time"
+)
+
+const limit = 5
```

Comment:
Is this limit the time a user is connected? Then it should be named as such. Also, we started using direct `time.Duration` instead of `int`.

Reply:
Fixed.